eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -22,11 +22,20 @@
     <UsagePattern IdentityGlob="Microsoft.NETCore.App.Ref/7.0*" />
 
      <!-- This is upgraded to latest version in full source-build and can be baselined for repo build -->
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Configuration*/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.DependencyInjection/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Logging*/8.0*" />
+    <UsagePattern IdentityGlob="Microsoft.Extensions.Options.ConfigurationExtension*/8.0*" />
```

Comment:
This is missing from eng/Directory.Packages.props.

Reply:
Added. Let me double check if all these packages are added

Reply:
OK Found another one. Added.